packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -148,7 +148,7 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
         organization: {
           create: {
             name,
-            ...(IS_TEAM_BILLING_ENABLED ? { slug } : {}),
+            ...(!IS_TEAM_BILLING_ENABLED ? { slug } : {}),
```

Comment:
Unintentional change that broke org branding locally (when `IS_TEAM_BILLING_ENABLED` is turned off), noted in the original PR after the fact [here](https://github.com/calcom/cal.com/pull/10967/files#r1320775851)